site/documentation/content/api/event/index.md

```diff
@@ -23,7 +23,7 @@ The following operations can be used by *Protocol Adapters* to forward event mes
 **Preconditions**
 
 1. Adapter has established an AMQP connection with the AMQP Messaging Network.
-1. Adapter has established an AMQP link in role *sender* with the AMQP Messaging Network using target address `event/${tenant_id}` where `${tenant_id}` is the ID of the tenant that the client wants to upload event messages for. 
+1. Adapter has established an AMQP link in role *sender* with the AMQP Messaging Network using target address `event/${tenant_id}` where `${tenant_id}` is the ID of the client for which the client wants to forward event messages.
```

Comment:
It was correct before FMPOV and is now changed for the worse as the tenant ID is definitely not the ID of the client.
Please revert this change.